docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -269,10 +269,41 @@ CertificateCredential credential = identity.getCredential(CertificateCredential.
 X509Certificate certificate = credential.getCertificate();
 ----
 
-==== Authorization
+[[map-certificate-attributes-to-roles]]
+==== Mapping certificate attributes to roles
+
+The information from the client certificate can be used to add roles to Quarkus `SecurityIdentity`.
+
+You can add new roles to `SecurityIdentity` after checking a client certificate's common name(CN) attribute.
+The easiest way to do is to use a certificate attribute to role mapping feature.
```

Comment:
Not sure whether it's just my English or a typo. the easiest to do WHAT?

Reply:
@michalvavrik I missed `it`, will add it